tests/baselines/reference/keyofAndIndexedAccess2.types

```diff
@@ -201,7 +201,7 @@ function f3<K extends string>(a: { [P in K]: number }, b: { [key: string]: numbe
 }
 
 function f3b<K extends string>(a: { [P in K]: number }, b: { [P in string]: number }, k: K) {
->f3b : <K extends string>(a: {    [P in K]: number;}, b: {    [P_1 in string]: number;}, k: K) => void
+>f3b : <K extends string>(a: { [P in K]: number; }, b: { [P in string]: number; }, k: K) => void

```

Comment:
👀 Do we have a bug re: `P_1` here?

Reply:
Yes but no - the bug is that P_1 is used at all - both the written types use `P`. The node builder is currently a bit too conservative in it's renaming rules for type parameters. Something I'm working on fixing separately.